UPGRADING.md

```diff
@@ -0,0 +1,134 @@
+# 2.0.0 Migration Guide
+
+The 2.0 release of the `google-cloud-firestore` client is a significant upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python), and includes substantial interface changes. Existing code written for earlier versions of this library may require updates to use this version. This document describes the changes that have been made, and what you need to do to update your usage.
+
+If you experience issues or have questions, please file an [issue](https://github.com/googleapis/python-firestore/issues).
+
+## Supported Python Versions
+
+> **WARNING**: Breaking change
+
+The 2.0.0 release requires Python 3.6+.
+
+
+## Method Calls
+
+> **WARNING**: Breaking change
+
+If you previously were using modules or functions under the namespace
+`google.cloud.firestore_v1.gapic` there is a high likelihood you have incompatible code.
+To assist with this, we have includes some helpful scripts to make some of the code
+modifications required to use 2.0.0.
+
+* Install the library
+
+```py
+python3 -m pip install google-cloud-firestore
+```
+
+* The scripts `fixup_firestore_v1_keywords.py` and `fixup_firestore_admin_v1_keywords.py` 
+is shipped with the library. It expects an input directory (with the code to convert)
+and an empty destination directory.
+
+```sh
+$ fixup_firestore_v1_keywords.py --input-directory .samples/ --output-directory samples/
+$ fixup_firestore_admin_v1_keywords.py --input-directory .samples/ --output-directory samples/
```

Comment:
Nit: the `$` at the front of each line mean both lines aren't easily copy-pasteable at the same time and able to be pasted and ran in the shell.

Reply:
Didn't think about it but this may be a feature. Since the scripts don't do in place replacements, you are going to likely want to move things about between.